src/gcp/apphosting.ts

```diff
@@ -334,6 +347,33 @@ export async function getBuild(
   return res.body;
 }
 
+/**
+ * List Builds by backend
+ */
+export async function listBuilds(
+  projectId: string,
+  location: string,
+  backendId: string,
+): Promise<ListBuildsResponse> {
+  const name = `projects/${projectId}/locations/${location}/backends/${backendId}/builds`;
+  let pageToken: string | undefined = undefined;
```

Comment:
Technically I don't think you need to specify `= undefined` - that can be dropped.

```suggestion
  let pageToken: string | undefined;
```